test/browser/specs/inspect.test.ts

```diff
@@ -0,0 +1,114 @@
+import type { InspectorNotification } from 'node:inspector'
+import { expect, test, vi } from 'vitest'
+import WebSocket from 'ws'
+
+import { runVitestCli } from '../../test-utils'
+
+type Message = Partial<InspectorNotification<any>>
+
+const REMOTE_DEBUG_URL = '127.0.0.1:9123'
+
+test('--inspect-brk stops at test file', async () => {
+  const { vitest, waitForClose } = await runVitestCli(
+    '--root',
+    'fixtures/inspect',
+    '--browser',
+    '--no-file-parallelism',
+    '--inspect-brk',
+    REMOTE_DEBUG_URL,
+  )
```

Comment:
This test is based on Node pool's equivalent, but adding browser mode tests there didn't look right.

https://github.com/vitest-dev/vitest/blob/8d883cf05832241e6d8f97d099f6d1fdd4e231d5/test/cli/test/inspect.test.ts#L1-L92